src/platforms/android/configuration/integrations/jetpack-compose.mdx

```diff
@@ -8,13 +8,66 @@ categories:
   - mobile
 ---
 
-The `sentry-compose` library provides [Jetpack Compose](https://developer.android.com/jetpack/androidx/releases/compose) Navigation support for Sentry.
+The `sentry-compose` library provides [Jetpack Compose](https://developer.android.com/jetpack/androidx/releases/compose) support for the following interactions:
+* User interactions like clicks, swipe and scroll gestures
+* Jetpack Compose navigation
+
+## User Interactions
```

Comment:
I think we should also mention here the gradle setup (because it requires `sentry-compose` in the dependencies, right?). E.g. this is for navigation:

### Manual Installation

#### Install

Sentry captures data by adding a `withSentryObservableEffect` extension function to `NavHostController`. To add the Navigation integration, install the [Android SDK](/platforms/android/), then add the `sentry-compose` dependency using Gradle:

```groovy
implementation 'io.sentry:sentry-android:{{ packages.version('sentry.java.android', '6.2.0') }}'
implementation 'io.sentry:sentry-compose-android:{{ packages.version('sentry.java.compose.android', '6.2.0') }}'
```